app/views/landing-pages/roblox/PageRoblox.vue

```diff
@@ -1,100 +1,6 @@
 <template>
   <div id="roblox-page">
     <div class="container-fluid headline-container">
-      <div class="responsive-background">
```

Comment:
why the removal?

Reply:
what? i don't think i change the page roblox... strange ...